test/app_test.rb

```diff
@@ -1,15 +1,191 @@
 require "test_helper"
 
 class AppTest < Minitest::Test
+  def setup
+    @app = Minesweeper_2pl::App.new
+    @mocked_app = Minesweeper_2pl::App.new
+    @mocked_game = Minesweeper_2pl::MockedGame.new
+    @mocked_cli = Minesweeper_2pl::MockedCli.new
+    @mocked_app.game = @mocked_game
+    @mocked_app.cli = @mocked_cli
+    @mocked_app.game.setup(100,100)
+  end
 
   def test_that_it_has_an_app_class
-    app = Minesweeper_2pl::App.new
-    refute_nil app
+    refute_nil @mocked_app
+  end
+
+  def test_that_it_has_a_cli
+    refute_nil @mocked_app.cli
```

Comment:
This is OK, but the assertion is pretty weak. If you want to check that your `App` has access to a `CLI`, is there a stronger assertion you can make? Is there a way to access an instance's class name in Ruby? Maybe a helper method in minitest?